tests/test_plain_text_to_pandoc.py

```diff
@@ -21,8 +21,3 @@ def test_plain_text_to_pandoc_spaces_after():
 def test_plain_text_to_pandoc_spaces_after_newline():
     pandoc_ast = RichText.plain_text_to_pandoc("hello\n  world")
     assert pandoc_ast == [Str("hello"), LineBreak(), Space(), Space(), Str("world")]
-
-
-def test_plain_text_to_pandoc_table():
```

Comment:
I assume you're deleting this test case because it breaks after your changes? Let's fix the test case so that it works with your changes.

Reply:
I created it as a test case for the previous changes